06.bowling_object/game.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  LAST_FRAME = 9
+
+  def initialize(marks)
+    @frames = []
+    score_idx = 0
+    0.upto(LAST_FRAME) do |frame_idx|
+      frame = Frame.new(frame_idx)
+      score_idx = frame.add_shot(marks, score_idx)
```

Comment:
```suggestion
      score_idx += frame.add_shot(marks[score_idx..-1])
```

- `score_idx`はGameクラスが管理する
- Frameクラスはscore_idxの存在を気にせず、引数で渡されたmarksを前から順番に追加していく
- `add_shot`は追加したmarkの件数を返す

という設計にした方が互いの責務が明確になっていいかな、と思いました。